test/statsCases/performance-oversize-node-target-no-hints/webpack.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+	mode: "production",
+	entry: "./index",
+	target: "node",
+	stats: {
+		colors: true,
+		hash: false,
+		entrypoints: true
+	}
+};
```

Comment:
As we are trying to fix the original broken logic, we should add some test cases to verify it.

You can use an array of configurations here to test multiple configuration in a single test case.

It would be good to test the defaults for `mode` development and production.